lab-jonathan/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "lab-jonathan",
+  "version": "1.0.0",
+  "description": "lab 11 express",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "dependencies": {
+    "bluebird": "^3.4.6",
```

Comment:
Need to refactor these modules... you have a ton of stuff in here that should be under dev deps.